src/protocols/vnc/settings.c

```diff
@@ -351,6 +353,12 @@ enum VNC_ARGS_IDX {
      * using the clipboard. By default, clipboard access is not blocked.
      */
     IDX_DISABLE_PASTE,
+
+    /*
```

Comment:
This should be `/**` (Doxygen-style comment).

Reply:
Fixed via rebase.